docs/diagrams/MatchCustomerSequenceDiagram.puml

```diff
@@ -0,0 +1,67 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":MatchCustomerCommandParser" as MatchCustomerCommandParser LOGIC_COLOR
+participant "command:MatchCustomerCommand" as MatchCustomerCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("matchcust 1")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("matchcust 1")
+activate AddressBookParser
+
+create MatchCustomerCommandParser
+AddressBookParser -> MatchCustomerCommandParser : new MatchCustomerCommandParser("1")
```

Comment:
Im not too sure if you're supposed to add 'new' or the parameters. Please correct me if I'm wrong!